src/components/List/AddCard/AddCardPrompt/AddCardPrompt.js

```diff
@@ -0,0 +1,21 @@
+import React, { useRef } from "react";
+import { FiPlus } from "react-icons/fi";
+
+const AddCardPrompt = ({ open, setOpen }) => {
+  const ref = useRef();
+  const inlineStyle = open ? { height: ref.current?.scrollHeight } : { height: 0, overflow: "hidden" };
```

Comment:
Like above, this inlineStyle controls the visibility of the element. If it isnt open, the height is set to 0